routes/account/upgrade.ts

```diff
@@ -28,6 +28,6 @@ export const handler: Handlers<null, AccountState> = {
       mode: "subscription",
     });
 
-    return redirect(url);
+    return redirect(url || "/");
```

Comment:
Can you please revert this change?

Reply:
Sure

it triggers this error though when running `deno task ok` on local machine

```
error: TS2345 [ERROR]: Argument of type 'string | null' is not assignable to parameter of type 'string'.
  Type 'null' is not assignable to type 'string'.
    return redirect(url);
                    ~~~
    at file:///....../...../saaskit/routes/account/upgrade.ts:31:21
```

Reply:
@huai-jie, if you're still able to reproduce this issue, could you please create a GitHub issue?